src/threshold-ecdsa/threshold-ecdsa-construction/introduction.md

```diff
@@ -0,0 +1,12 @@
+## Gennaro's Construction
+
+In this section we briefly describe the threshold ECDSA protocol of Gennaro etal in {{#cite GG20}}, in which we assume that the readers have some familiarity to ECDSA signature scheme. Recall that the ordinary ECDSA signature scheme \\(\sigma=(r,s)\\) of a message \\(M\\) is generated as follow
+
+$$r=R.\mathsf{x},~R=g^{k^{-1}},~m=H(M)~\text{and}~s=k(m+r\cdot sk)$$
```

Comment:
`H()` is collision resistance hash function, it shouldn't be considered as a part of ECDSA.

Reply:
`s=k(m+r⋅sk)` or `s = k^{-1}(m+r⋅sk)`?

Reply:
`R=g^{k^{-1}}` or `R=g^{k}`?

Reply:
s=k(m+r⋅sk) and R=g^{k^{-1}}. The version of threshold ecdsa use k^{-1} instead of k. It does not change the verification process anyway.